pkg/apb/rhcc_registry.go

```diff
@@ -57,19 +58,17 @@ func (r RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
 	r.log.Debug("RHCCRegistry::LoadSpecs")
 	var specs []*Spec
 
-	imageList, err := r.LoadImages("*-apb")
+	imageList, err := r.LoadImages("\"*-apb\"")
 	if err != nil {
 		return []*Spec{}, 0, err
 	}
 
 	numResults := imageList.NumResults
 	r.log.Debug("Found %d images in RHCC", numResults)
 	for _, image := range imageList.Results {
-		spec, err := r.imageToSpec(image)
-		if err != nil {
-			return []*Spec{}, 0, err
+		if spec, err := r.imageToSpec(image); err == nil {
+			specs = append(specs, spec)
```

Comment:
Since we're adding valid images and skipping invalid ones, should the log that identifies an invalid image be ```log.Info``` instead of ```log.Error```? I think it's great we identify there's an issue with an apb, but it's not a broker error.